Misc/NEWS.d/next/Security/2021-03-13-21-25-29.bpo-43223.ieBVWq.rst

```diff
@@ -0,0 +1,2 @@
+:mod:`http.server`: Fix an open redirection vulnerability in the HTTP server when an URL contains ``//`.

```

Comment:
Missing a second backtick after the `//` which caused the doc build CI test to fail.

Reply:
It should pass the test now.